### ISSUE TITLE
Possible error in Combinations and Permutations exercise 8 hint

### ISSUE THREAD

**Issue by morbidCode** [oscarlevin/discrete-book#127]
Hi. I was currently doing exercise 8 of Combinations and Permutations here (https://discrete.openmathbooks.org/dmoi3/sec_counting-combperm.html).
The exercise is: "How many anagrams are there of the word “assesses” that start with the letter “a”?"
There is a hint: "We just need a string of 7 letters: 4 of one type, 3 of the other."
Since “assesses” has 5 ss and 2 es, should the hint be: "We just need a string of 7 letters: 5 of one type, 2 of the other."?
Thanks.

**Comment by oscarlevin** [comment]
Yeah!  Nice catch.  Thanks for the report.  I'll get it fixed soon.
